pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Overview:
```

Comment:
Add boilerplate thanking the user for their contribution, and link to our contributing guidelines.

Reply:
added contribution thank you

Reply:
Please also add a link to our contribution guidelines.

Reply:
I added one in "checklist" section but I'll add one at the top as well.